tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc

```diff
@@ -680,13 +680,20 @@ TEST (nnstreamer_nnfw_mlapi, invoke_single_02_n)
 #define wait_for_sink(expected_cnt)                 \
   do {                                              \
     guint waiting_time = 0;                         \
-    while (*sink_called_cnt < expected_cnt) {       \
-      g_usleep (TEST_DEFAULT_SLEEP_TIME);           \
+    while (1) {                                     \
+      gboolean exit = FALSE;                        \
+      g_mutex_lock (&g_test_mutex);                 \
+      exit = (*sink_called_cnt >= expected_cnt);    \
       waiting_time += TEST_DEFAULT_SLEEP_TIME;      \
+      g_mutex_unlock (&g_test_mutex);               \
+      if (exit) break;                              \
+      g_usleep (TEST_DEFAULT_SLEEP_TIME);           \
```

Comment:
Do not rely on assert for behaviors. It should behave correctly even ASSERT is ignored.

Reply:
`ASSERT_RT` was removed. Thanks.

Reply:
No. Do not remove it. It's a valid assertion.

You need to make it work even if ASSERT is ignored by compilers.

Reply:
Got it. I'll change `EXPECT_TRUE` to `ASSERT_TRUE`. The tests will work without this assert.